testnet-contracts/migrations/2_next.js

```diff
@@ -1,7 +1,46 @@
-var Peggy = artifacts.require("Peggy");
-var TestToken = artifacts.require("TestToken");
+const Valset = artifacts.require("Valset");
+const CosmosBridge = artifacts.require("CosmosBridge");
+const Oracle = artifacts.require("Oracle");
+const BridgeBank = artifacts.require("BridgeBank");
 
 module.exports = function(deployer, network, accounts) {
-  deployer.deploy(TestToken, { gas: 4612388, from: accounts[0] });
-  deployer.deploy(Peggy, { gas: 4612388, from: accounts[0] });
+  const operator = accounts[0];
+  const initialValidators = [accounts[1], accounts[2], accounts[3]];
+  const initialPowers = [5, 8, 12];
```

Comment:
This migration is very specific in terms of initial validators/powers. Ideally this should be paramaterized or taken from a config file that the end user can configure themself. Maybe make our own config file or can just add some fields to the truffle config to use.

Reply:
https://github.com/cosmos/peggy/pull/82/commits/7bd1e59626ea28671a880bdadd079baa071905f4, https://github.com/cosmos/peggy/pull/82/commits/b4109a915f03e594aa0285a125f39a1070d00a46